ddp-workspace/projects/ddp-sdk/src/lib/services/renewSessionNotifier.service.ts

```diff
@@ -3,12 +3,12 @@ import { DOCUMENT } from '@angular/common';
 import { Title } from '@angular/platform-browser';
 import { NGXTranslateService } from './internationalization/ngxTranslate.service';
 import { CompositeDisposable } from '../compositeDisposable';
-import { Subject, Observable } from 'rxjs';
+import { ReplaySubject, Subject, Observable } from 'rxjs';
 
 @Injectable()
 export class RenewSessionNotifier implements OnDestroy {
     private anchor: CompositeDisposable = new CompositeDisposable();
-    private openDialog: Subject<void> = new Subject<void>();
+    private openDialog: ReplaySubject<void> = new ReplaySubject<void>(1);
```

Comment:
Found a bug:
1. Session dialog was shown 
2. A user reloads a page 
3. In half the cases, dialog won't be shown after page reloading. 
I suppose the reason for the bug was that we subscribed to `openDialogEvents` after the event happened, so we heed to save the last event.